content/en/docs/troubleshooting.md

```diff
@@ -1,6 +1,88 @@
+
+```shell
+notation cert show --type ca --store mytruststore mycertificate.crt
+```
+
+An example of output:
+
+```text
+Certificate details
+--------------------------------------------------------------------------------
+Issuer: CN=example.io,O=Notary,L=Seattle,ST=WA,C=US
+Subject: CN=example.io,O=Notary,L=Seattle,ST=WA,C=US
+Valid from: 2023-01-15 07:55:01 +0000 UTC
+Valid to: 2023-01-16 07:55:01 +0000 UTC
+IsCA: false
+SHA1 Thumbprint: xxx
+```
+
+Check the `Subject` info in the output. If it is the identity that signs the artifact, you need to add the `Subject` info into `trustedIdentities`.
+
+## I have configured trust policy, but I still get the error 'no applicable trust policy'
+
+This error indicates that the `registryScopes` property is not correctly configured. This property contains a list of repository URIs, where the artifacts are stored. You need to make sure the signing artifact is stored in one of the listed repositories. If not, you need to add the missing repository URI in `registryScopes`, or you can add a new trust policy for the missing repository.
+
+The repository URI is in the format of `${registry-name}/${namespace}/${repository-name}`. For example, if the artifact to be verified is `registry.acme-rockets.io/software/net-monitor@sha256:xxx`, then the value for `registryScopes` should be `registry.acme-rockets.io/software/net-monitor`, the following values are wrong
+
+- `registry.acme-rockets.io`
+- `registry.acme-rockets.io/software`
+
+## When I verify an artifact, I get the error 'malformed trustpolicy.json file'
+
+This is normally an encoding problem of `trustpolicy.json` file. Notation expects `utf-8 without BOM` or `ascii` encoding for `trustpolicy.json` file.
+
+For Windows user, Windows PowerShell (prior to v6) uses the Unicode `UTF-16LE` encoding by default, and `utf-8 without BOM` is not supported. If you are building `trustpolicy.json` file in Windows PowerShell (prior to v6), make sure you change the file encoding to `ascii`.
+
+## When I verify an artifact, I get the error 'Failed to unmarshal the payload content in the signature blob to envelope.Payload'
+
+This is normally an encoding problem of payload content in the signature envelope. Notary v2 signatures could be produced by different tools per [signature specification](https://github.com/notaryproject/notaryproject/blob/v1.0.0-rc.2/specs/signature-specification.md). The payload content is a `JSON` document defined in the signature specification, and the encoding should be `utf-8 without BOM` or `ascii`.
```

Comment:
Notary v2 should not be used anywhere in the documentation. We should change to "Notary signatures"